src/main/java/greencity/entity/Comment.java

```diff
@@ -33,10 +33,12 @@
     @ManyToOne
     private Comment parentComment;
 
-    @OneToMany(mappedBy = "parentComment", fetch = FetchType.LAZY, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @OneToMany(mappedBy = "parentComment", fetch = FetchType.LAZY,
+        cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.ALL})
     private List<Comment> comments = new ArrayList<>();
 
-    @OneToMany(mappedBy = "comment", fetch = FetchType.LAZY, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @OneToMany(mappedBy = "comment", fetch = FetchType.LAZY,
```

Comment:
fetch  type lazy is by default and return cascade type

Reply:
Dear Marian, i fix this problem